recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -83,40 +83,46 @@ def requirements(self):
         self.requires("gemmlowp/cci.20210928")
         if self.settings.arch in ("x86", "x86_64"):
             self.requires("intel-neon2sse/cci.20210225")
-        self.requires("ruy/cci.20210622")
+        self.requires("ruy/cci.20220628")
         if self.options.with_xnnpack:
-            self.requires("xnnpack/cci.20211210")
+            self.requires("xnnpack/cci.20220621")
+
+        if self.options.with_xnnpack or self.options.get_safe("with_nnapi", False):
             self.requires("fp16/cci.20210320")
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[^3.16]")
```

Comment:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-are-version-ranges-not-allowed

We suggest just setting it to the latest version when a newer one is required

Reply:
done!